### DISCUSSION TITLE
PDF/A

### DISCUSSION THREAD

**Discussion by jamesonTT** [pymupdf/PyMuPDF#1112] [Q&A] [answered]
Hiya, when combing multiple PDFs into a single file, is it possible to create the new PDF as PDF/A compliant? - we may also be receiving images we need to convert to a PDF first (which we would use this tool for also)

**Comment by JorjMcKie** [comment, marked as answer]
No, unfortunately this is not currently supported.
Try [ocrmypdf](https://github.com/jbarlow83/OCRmyPDF) for this. Maybe also in combination with PyMuPDF.

**Reply by jamesonTT** [reply]
Thanks @JorjMcKie - apologies for the tardy reply (and much appreciate how quick your response was).
JT
